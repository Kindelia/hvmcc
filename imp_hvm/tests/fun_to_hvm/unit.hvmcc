function test_unit () {
    {test_unit Unit} = Unit
}