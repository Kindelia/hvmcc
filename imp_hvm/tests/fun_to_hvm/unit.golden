(test_unit (Unit)) = (Unit)